website/docs/r/budgets_budget.html.markdown

```diff
@@ -58,6 +58,48 @@ resource "aws_budgets_budget" "s3" {
 }
 ```
 
+Create a Savings Plan Utilization Budget
+```hcl
+resource "aws_budgets_budget" "ri_utilization" {
+  # ...
+  budget_type = "SAVINGS_PLANS_UTILIZATION"
+  limit_amount = "100"
+  limit_unit = "PERCENTAGE"
+
```

Comment:
While we're fixing things in here could you apply a round of `terraform fmt` on these (lining up the equals signs and removing extraneous ending new lines like this one here? Thanks!

```suggestion
```